gqueries/general/electricity/electricity_curtailment.gql

```diff
@@ -0,0 +1,15 @@
+- query =
+  DIVIDE(
+    SUM(
+      V(
+        energy_flexibility_curtailment_electricity,
+        energy_hydrogen_curtailed_electricity,
+        energy_power_hybrid_curtailed_electricity,
+        energy_battery_curtailed_solar_electricity,
+        energy_battery_curtailed_wind_electricity,
```

Comment:
I don't think these should be included. If we only show the **output** of the integrated battery systems **to the grid** in the mekko, then we do not have to include the curtailment right?